de1plus/vars.tcl

```diff
@@ -1104,7 +1104,7 @@ proc drink_weight_text {} {
 		return ""
 	}
 
-	return [return_weight_measurement $::settings(drink_weight)]
+	return [return_weight_measurement [ifexists ::settings(running_weight) 0.0]]
```

Comment:
Would it be better to initialize `::settings(running_weight)` in `machine.tcl` so that a run-time check does't need to be made on every call?

Reply:
> Would it be better to initialize `::settings(running_weight)` in `machine.tcl` so that a run-time check doesn't need to be made on every call?

You're right. Changing it.